### ISSUE TITLE
telegraf 0.2.3 and plugin procstat

### ISSUE THREAD

**Issue by logicall** [influxdata/telegraf#510]
Hello!
I run telegraf on rhel 5.8 x32. And uses a plugin protsstat.

The database get weird data

```
name: procstat_notification_server_cpu_user
-------------------------------------------
time			host		name	pid	value
1452603440000000000	va3 python	32620	9215.53
1452603450000000000	va3	python	32620	9215.85
1452603460000000000	va3	python	32620	9215.99
```

On rhel 6.3 x64 this plugin works fine and displays and the normal values.

```
name: procstat_callapi_9041_cpu_user
------------------------------------
time			host		name	pid	value
1449062470000000000	app	python	212606	17.92
1449062480000000000	app	python	212606	17.92
1449062490000000000	app	python	212606	17.92
```

Tell me, because of what this could be?

**Comment by sparrc** [comment]
The `procstat` plugin is reporting cpu _time_ instead of cpu usage. See https://github.com/influxdata/telegraf/issues/484

In 0.10.0 the metric name has been changed to include `_time`

**Comment by sparrc** [comment]
closing as won't fix, cpu usage will come with #484
